website/templates/project/modal_add_contributor.mako

```diff
@@ -137,8 +137,8 @@
                                 <thead data-bind="visible: selection().length">
                                     <th width="10%"></th>
                                     <th width="15%"></th>
-                                    <th width="45%">Name</th>
-                                    <th width="30%">
+                                    <th>Name</th>
```

Comment:
Width is deprecated, the suggested way is to use css for this. http://help.simplytestable.com/errors/html-validation/the-x-attribute-on-the-y-element-is-obsolete-use-css-instead/the-width-attribute-on-the-td-element-is-obsolete-use-css-instead/

However we don't follow this overall on the site, so it's not a blocker.